code/game/objects/items/weapons/RCD.dm

```diff
@@ -152,8 +152,13 @@ RCD
 				var/obj/item/pda/pda = I
 				I = pda.id
 			var/obj/item/card/id/ID = I
-			if(istype(ID) && ID && check_access(ID))
-				locked = 0
+			if(istype(ID) && ID)
+				if(check_access(ID))
+					locked = 0
+				else 
+					to_chat(user, "<span class='warning'>This ID lacks access.</span>")

```

Comment:
You'll need `usr` or to cast to `user` before this.

Reply:
yeah did not save the changes I made when copying. Fixed now